config/database.yml.travis

```diff
@@ -0,0 +1,21 @@
+default: &default
+  adapter:  postgresql
+  encoding: unicode
+  pool:     5
+  timeout:  5000
+
+development:
+  <<: *default
+  database: mahonia_development
+  username: mahonia
+  password: password1
+
+test:
+  <<: *default
+  database: travis_ci_test
```

Comment:
I'm not sure why we're not just saying `database: mahonia_test` on line 15 and checking a single `config/database.yml` into source control.  I'm not getting what benefit or flexibility this gives us.  And it seems sort of complex.

Reply:
This is Travis's suggested configuration approach. See: https://docs.travis-ci.com/user/database-setup/#PostgreSQL